config/speechd.conf

```diff
@@ -197,16 +197,23 @@ SymbolsPreprocFile "symbols-fallback.dic"
 #       "pulse" - PulseAudio
 #       "alsa"  - Advanced Linux Sound System
 #       "oss"   - Open Sound System
+#        "pipewire" -   the new low-latency sound server, for near realtime speech performance
 #       "nas"   - Network Audio System
 #       "libao" - A cross platform audio library
-# Pulse audio is the default and recommended sound server. OSS and ALSA
+# Pipe Wire is the newest audio output method in this configuration, but not the default one.
+# However, if it works in your deployment, it is recommended because of the latency related advantages it has over
+# pulseaudio and others,
+#however enabling it should be done with precaution, because it isn't  tested to work in all configurations supported by speech dispatcher.
+#For now, enabling this is at the discression of the users and packagers of speech dispatcher. However, because we want to eventually make this the default for everyone, trying it in your deployments, if non-critical, is recommended
+# OSS, Pulse Audio and ALSA
 # are only provided for compatibility with architectures that do not
-# include Pulse Audio. NAS provides network transparency, but is not
-# very well tested. libao is a cross platform library with plugins for
+# include Pipe Wire. NAS provides network transparency, but is not
+# very well tested.
+# libao is a cross platform library with plugins for
 # different sound systems and provides alternative output for Pulse Audio
 # and ALSA as well as for other backends.
 
-# AudioOutputMethod "pulse"
+#audioOutputMethod "pipewire"
```

Comment:
Beware of the typo, and the removal of space. This file is used as template for spd-conf to produce the user configuration, so we absolutely need `AudioOutputMethod` to appear as such, and for a space to exist between `#` and `AudioOutputMethod`

Reply:
this has now been fixed. Thanks, didn't know spd-conf exists